ExampleApp/AppDelegate.swift

```diff
@@ -234,3 +234,44 @@ class DummyMarketDataService: MarketDataService {
         onFailure(error)
     }
 }
+
+// Only implement this protocol if you need to stream your own market data
+class DummyStreamingMarketDataService: StreamingMarketDataService {
```

Comment:
Is this on by default now in the SDK? If not, is there an easy way to test it from the ExampleApp? Maybe we can add a `Toggle Dummy streaming` button to the example app.